resources/i18n/it_IT/fdmextruder.def.json.po

```diff
@@ -68,41 +101,41 @@ msgctxt "extruder_prime_pos_z label"
 msgid "Extruder Prime Z Position"
 msgstr "Posizione Z innesco estrusore"
 
-msgctxt "machine_extruder_cooling_fan_number label"
-msgid "Extruder Print Cooling Fan"
-msgstr "Ventola di raffreddamento stampa estrusore"
+msgctxt "machine_extruder_end_pos_y description"
+msgid "The y-coordinate of the ending position when turning the extruder off."
+msgstr "La coordinata y della posizione di fine allo spegnimento dell’estrusore."
 
 msgctxt "machine_extruder_start_code label"
 msgid "Extruder Start G-Code"
 msgstr "Codice G avvio estrusore"
 
-msgctxt "machine_extruder_start_code_duration label"
-msgid "Extruder Start G-Code Duration"
-msgstr ""
+msgctxt "machine_extruder_start_code description"
+msgid "Start g-code to execute when switching to this extruder."
+msgstr "Inizio codice G da eseguire quando si passa a questo estrusore."
 
-msgctxt "machine_extruder_start_pos_abs label"
-msgid "Extruder Start Position Absolute"
-msgstr "Assoluto posizione avvio estrusore"
+msgctxt "extruder_prime_pos_x description"
+msgid "The X coordinate of the position where the nozzle primes at the start of printing."
+msgstr "La coordinata X della posizione in cui l’ugello si innesca all’avvio della stampa."
 
-msgctxt "machine_extruder_start_pos_x label"
-msgid "Extruder Start Position X"
-msgstr "X posizione avvio estrusore"
+msgctxt "extruder_prime_pos_y description"
+msgid "The Y coordinate of the position where the nozzle primes at the start of printing."
+msgstr "La coordinata Y della posizione in cui l’ugello si innesca all’avvio della stampa."
 
-msgctxt "machine_extruder_start_pos_y label"
-msgid "Extruder Start Position Y"
-msgstr "Y posizione avvio estrusore"
+msgctxt "extruder_prime_pos_z description"
+msgid "The Z coordinate of the position where the nozzle primes at the start of printing."
+msgstr "Indica la coordinata Z della posizione in cui l’ugello si innesca all’avvio della stampa."
 
-msgctxt "machine_settings label"
-msgid "Machine"
-msgstr "Macchina"
+msgctxt "machine_extruder_start_code_duration label"
+msgid "Extruder Start G-Code Duration"
+msgstr "Durata del G-code di inizio dell'estrusore"
 
-msgctxt "machine_settings description"
-msgid "Machine specific settings"
-msgstr "Impostazioni macchina specifiche"
+msgctxt "machine_extruder_start_code_duration description"
+msgid "The time it'll take to execute the start g-code, when switching to this extruder."
+msgstr "Il tempo necessario per eseguire il G-code di inizio, quando si passa a questo estrusore."
 
-msgctxt "machine_extruder_end_pos_abs description"
-msgid "Make the extruder ending position absolute rather than relative to the last-known location of the head."
-msgstr "Rende la posizione di fine estrusore assoluta anziché relativa rispetto all’ultima posizione nota della testina."
+msgctxt "machine_extruder_start_pos_abs label"
+msgid "Extruder Start Position Absolute"
+msgstr "Assoluto posizione avvio estrusore"
```

Comment:
Not sure I get it correctly but it may translate to "Posizione assoluta avvio estrusore" if it means that this is an absolute position. By the way position is female it it's the subject it should translate to "assoluta"

Reply:
Well the description of the setting says "Make the extruder starting position absolute rather than relative to the last-known location of the head." so I assume you are right.